src/app/containers/RadioPageBlocks/Blocks/AudioPlayer/index.jsx

```diff
@@ -29,22 +33,17 @@ const getMediaInfo = (assetId) => ({
 const getMasterBrand = (masterBrand, liveRadioIdOverrides) =>
   pathOr(masterBrand, ['masterBrand', masterBrand], liveRadioIdOverrides);
 
-const OuterWrapper = styled.div`
-  @media (min-width: 63rem) {
-    display: flex;
-    justify-content: center;
+const AudioPlayerWrapper = styled.div`
+  width: calc(100% + ${GEL_SPACING_DBL});
+  margin-left: -${GEL_SPACING};
+  @media (min-width: 25rem) {
```

Comment:
We have a variable for this: https://github.com/bbc/psammead/blob/latest/packages/utilities/gel-foundations/src/breakpoints.js#L12